packages/components/src/Banner/Banner.css

```diff
@@ -30,6 +63,16 @@
   flex: 0;
 }
 
+/* .medium .closeButton {
+  top: var(--flash--inset);
+} */
+
+.closeButton:hover {
+  background: rgba(0, 0, 0, 0.025);
+}
+
 .closeButton:focus {
+  box-shadow: 0 0 1px 0.125rem var(--color-outline);
```

Comment:
Is there a reason that we didnt use the `--shadow-base` here?

Reply:
`--shadow-base` is for things that are meant to have a "shadow" appearance to indicate depth/elevation (toast, menu, etc), where this is simulating an outline. We don't use actual outline because our current style across the product has more of a "glow" than default outline options.

Reply:
Do you think it's worth consolidating this style in an `--shadow-outline` variable though??

Reply:
if you think that we will use this pattern again, then yes i think so.